src/docs/get-started/editor.md

```diff
@@ -74,21 +74,20 @@ To install these:
 
 ## Install VS Code
 
-VS Code is a lightweight editor with Flutter app execution and debug support.
+VS Code is a lightweight editor with Flutter app execution and debugs support.
 
 * [VS Code][], latest stable version
 
 ## Install the Flutter and Dart plugins
 
  1. Start VS Code.
- 1. Invoke **View > Command Palette...**.
+ 1. Invoke **View > Command Palette...**
  1. Type "install", and select **Extensions: Install Extensions**.
- 1. Type "flutter" in the extensions search field, select **Flutter** in the list,
-    and click **Install**. This also installs the required Dart plugin.
+ 1. Type "flutter" in the extensions search field, select **Flutter** in the list and click **Install**. This also installs the required Dart plugin.
```

Comment:
It appears that the only change here is to make a super long line. We break our lines at 80 chars OR LESS. (Often less, because we break on phrasing.)